ansible/roles/software/ckan/inventory/tasks/install.yml

```diff
@@ -31,10 +31,19 @@
 - name: Remove new code
   file: path="{{ project_source_new_code_path }}" state=absent
 
-- include: build-pkgs.yml
-  vars:
-    pkg: "{{ ckan_pkgs }}"
-    virtualenv: "{{ project_source_new_code_path }}"
+- name: Clone Inventory app
+  git:
+    repo: '{{ app_repo }}'
+    dest: '{{ app_source_path }}'
+    version: '{{ app_repo_branch }}'
```

Comment:
Would you please add a `umask: "0022"` to make sure these files are world-readable.

Reply:
Added requested changes